src/components/ContentSidebar/SidebarVersions.js

```diff
@@ -27,9 +26,9 @@ const SidebarVersions = ({
     }
 
     return (
-        <SidebarSection>
+        <div>
```

Comment:
Need a wrapping div? Can we just return the component itself (and internally add block style to it if needed)